sql/src/test/java/io/crate/executor/transport/ddl/TransportRenameTableActionTest.java

```diff
@@ -62,21 +62,23 @@ public void testRenameOnOpenTableThrowsException() throws Exception {
 
     @Test
     public void testRenameOnOpenPartitionedTableThrowsException() throws Exception {
-        RenameTableRequest request = new RenameTableRequest(TableIdent.fromIndexName("p1"),
-            TableIdent.fromIndexName("p2"), true);
+        String defaultSchema = sqlExecutor.getDefaultSchema();
```

Comment:
Do you think it would make sense to provide a method that returns the full qualified name of a table/index (including the randomized schema)?